packages/xo-server/src/api/pool.mjs

```diff
@@ -131,9 +132,14 @@ installPatches.description = 'Install patches on hosts'
 
 // -------------------------------------------------------------------
 
-export async function rollingUpdate({ pool }) {
+export const rollingUpdate = deferrable(async function ($defer, { pool }) {
+  if ((await this.getPlugin('load-balancer'))?.loaded) {
```

Comment:
Does `getPlugin` also get the plugin if you pass the name?

Reply:
What do you mean?

Reply:
`load-balancer` is both the plugin's `id` and `name`.